pkg/sensors/program/loader.go

```diff
@@ -804,8 +805,33 @@ func doLoadProgram(
 		}
 	}
 
+	// We have following maps available for loading:
+	// - maps attached/pinned to program directly in load.PinMap[name]
+	// - maps passed to loader
+	//
+	// We take user maps from loadOpts.Maps and use them to resolve
+	// program maps in following order:
+	//   1) load.PinMap[name]
+	//   2) loadOpts.Maps user maps
+
+	maps := map[string]*Map{}
+	for _, pm := range loadOpts.Maps {
+		if !pm.IsOwner() {
```

Comment:
Out of curiosity, why are we skipping the "owned" maps?

Reply:
so the program loader now gets all the sensor maps to it's possible to resolve 'not owned' maps,
so we can use easily other sensor maps via constructs like:
`
maps = append(maps, program.MapUserFrom(base.ExecveMap))
`
I tried to explain that in the comment above this code, I'll try to put more details in there

Reply:
Thanks!